### PR TITLE
Don't use IPv6 on travis-ci

### PR DESCRIPTION
**Pull request by mfussenegger**
Trying to make Mr. Travis happy

### CODE REVIEW

File: sql/src/test/java/io/crate/integrationtests/PostgresITest.java [hunk 1]
```diff
@@ -65,7 +67,7 @@
 
     @BeforeClass
     public static void setupClass() {
-        useIPv6 = randomBoolean();
+        useIPv6 = randomBoolean() && !"true".equalsIgnoreCase(System.getenv(NO_IPV6));
```
Comment (matriv) [inline sql/src/test/java/io/crate/integrationtests/PostgresITest.java]: would put the property check first.

Comment (mfussenegger) [reply sql/src/test/java/io/crate/integrationtests/PostgresITest.java]: why?

Comment (matriv) [reply sql/src/test/java/io/crate/integrationtests/PostgresITest.java]: to avoid calling randomBoolean() unnecessarily

Comment (mfussenegger) [reply sql/src/test/java/io/crate/integrationtests/PostgresITest.java]: This is called once per job and a couple of ns at most? Might be that equalsIgnoreCase and `System.getenv` call is more expensive

**Pull request merged**